docs/http.md

```diff
@@ -481,6 +487,13 @@ step.
   direction of travel immediately after the maneuver.
 - `type` A string indicating the type of maneuver. **new identifiers might be introduced without API change**
    Types  unknown to the client should be handled like the `turn` type, the existance of correct `modifier` values is guranteed.
+- `lanes`: the available turn lanes at the turn
+  - `indication`: markings on the road, following the OSM scheme (e.g. left, slight_right, or through). Markings mey be combined into a list separated by ';'.
+  - `valid`: a boolean flag indicating whether the lane is a valid choice in the maneuver
+
+Currently, the supported lane tags are: `sharp_left, left, slight_left, sharp_right, right, slight_right, through, reverse, none`.
```

Comment:
In my current rewrite I already have the variation without underscore.
I feel that `all` is not a valid representation, because even if there is no marking (which would be all), it may only be allowed to go straight. The markings here represent the actual markings on the street.

The same way I would feel badly about direction. What we report here is nothing more than an indication, which is the current name. Direction would mean that it can be used only for a specific direction. A slight-right arrow can be used for exiting a round-about in two ways, for example.
We could go for type, but it would be confusing, since it is not a turn-road. So type would feel to me that it is a dedicated (separated) road. But as I said, indication is all we get.